src/operator/mxnet_op.cc

```diff
@@ -0,0 +1,31 @@
+/*
```

Comment:
remove file

Reply:
something is wrong with CI, it is building wrong commit..

Reply:
done